BigriverBookstore-apple/AppDelegate.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppDelegate.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Cocoa
+import Spine
+
+@NSApplicationMain
+class AppDelegate: NSObject, NSApplicationDelegate {
+
+    let baseURLpath = "http://jsonapiplayground.reyesoft.com"
+    
+
+    func applicationDidFinishLaunching(_ aNotification: Notification) {
+
+    }
+
```

Comment:
@c-Viorel If we have a method which does nothing and is not mandatory to implement them, my proposal is to delete the code. Less code, fewer bugs :)